pages/api/content/glamorous.js

```diff
@@ -1,6 +1,131 @@
 module.exports = {
-  title: '`glamorous`',
+  title: 'glamorous API',
   subtitle: '',
-  description: ``.replace(/~/g, '`'),
+  description: `
+    The ~glamorous~ function allows you to create your own ~glamorousComponentFactory~ (see 
+    [above](https://github.com/paypal/glamorous/blob/master/README.md#glamorouscomponentfactory)
+    ) for any component you have. For [~example~](https://codesandbox.io/s/g5kDAyB9:
```

Comment:
A typo in the link, it should be `[~example~](https://codesandbox.io/s/g5kDAyB9)`